op-challenger/fault/utils.go

```diff
@@ -0,0 +1,24 @@
+package fault
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type LogBuilder struct {
+	ctx []interface{}
+}
+
+// NewLogBuilder creates a new [logBuilder] instance.
+func NewLogBuilder() *LogBuilder {
```

Comment:
I'm not sure we need this functionality. We can also get a lot of this with `log.New`